sofiadinizms/vida-de-inseto
# Hotfix/final adjustments

## What I did
1. Final fru frus
2. Added cute assets
3. Added cut scenes in between levels
4. Added winning sound

## How to test
1. Pull and run this branch ON YOUR PHONE
2. Check if the app looks cuter and if it works fine
